js/src/features/project/epics.ts

```diff
@@ -171,7 +173,7 @@ export const fetchProjectRolePermissionsEpic: RootEpic = (
         map((result) =>
           result.success
             ? fetchProjectRoleAsync.success(result.data)
-            : fetchProjectRoleAsync.failure(result.errors),
+            : fetchProjectRoleAsync.failure(result.reason),
```

Comment:
You cannot just output reason. You should handle it and output your interpritation for this reason.
Again, `reason` - it is just constant.